distribution/docs/src/main/resources/content/_whitelists/catalog-whitelist.adoc

```diff
@@ -6,39 +6,167 @@
 :order: 01
 :summary: ${ddf-catalog} whitelist.
 
-The following packages have been exported by the ${ddf-catalog} application and are approved for use by third parties:
-
-* `ddf.camel.component.catalog`
-* `ddf.catalog`
-* `ddf.catalog.cache`
-* `ddf.catalog.data`
-* `ddf.catalog.data.metacardtype`
-* `ddf.catalog.event`
-* `ddf.catalog.federation`
-* `ddf.catalog.federation.impl`
-* `ddf.catalog.filter`
-* `ddf.catalog.filter.delegate`
-* `ddf.catalog.operation`
-* `ddf.catalog.plugin`
-* `ddf.catalog.plugin.groomer`
-* `ddf.catalog.pubsub`
-* `ddf.catalog.pubsub.tracker`
-* `ddf.catalog.resource`
-* `ddf.catalog.resource.data`
-* `ddf.catalog.resource.impl`
-* `ddf.catalog.resourceretriever`
-* `ddf.catalog.service`
-* `ddf.catalog.source`
-* `ddf.catalog.transform`
-* `ddf.catalog.transformer.api`
-* `ddf.catalog.transformer.metacard.geojson`
-* `ddf.catalog.util`
-* `ddf.catalog.validation`
-* `ddf.common`
-* `ddf.geo.formatter`
-* `ddf.util`
-* `org.codice.ddf.endpoints`
-* `org.codice.ddf.endpoints.rest`
-* `org.codice.ddf.endpoints.rest.action`
-* `org.codice.ddf.opensearch.query`
-* `org.codice.ddf.opensearch.query.filter`
+The following classes have been exported by the ${ddf-catalog} application and are approved for use by third parties:
+
+In package `ddf.catalog`
+
+* `CatalogFramework`
+* `Constants`
+
+In package `ddf.catalog.cache`
+
+* `ResourceCacheInterface` Deprecated
+* `SolrCacheMBean`
```

Comment:
New interface. Also an mbean so probably needs to be removed.

Reply:
I think that is being used by the solr commands to be able to clear the catalog metacard cache.

Reply:
@pklinef, does that mean remove?

Reply:
Talked to @jaymcnallie about this and it should not be included